src/System.Runtime.Extensions/src/System.Runtime.Extensions.csproj

```diff
@@ -191,6 +188,9 @@
     <Compile Include="$(CommonPath)\Interop\Windows\mincore\Interop.LookupAccountNameW.cs">
       <Link>Common\Interop\Windows\mincore\Interop.LookupAccountNameW.cs</Link>
     </Compile>
+    <Compile Include="$(CommonPath)\Interop\Windows\mincore\Interop.SHGetKnownFolderPath.cs">
```

Comment:
It is- both of these itemgroup blocks have the same condition. I'll merge the groups as I have to update the PR anyway.